chords_ble.py

```
@@ -72,10 +73,8 @@ def process_sample(self, sample_data: bytearray):
         if self.start_time is None:
             self.start_time = time.time()
         
-        channels = [
-            int.from_bytes(sample_data[1:3], byteorder='big', signed=True),
-            int.from_bytes(sample_data[3:5], byteorder='big', signed=True),
-            int.from_bytes(sample_data[5:7], byteorder='big', signed=True)]
+        channels = [int.from_bytes(sample_data[i:i+2], byteorder='big', signed=True) 
```

Ruff (0.11.9)

Local variable `channels` is assigned to but never used

Remove assignment to unused variable `channels`

(F841)